pkgs/development/python-modules/plexapi/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests, tqdm, websocket_client }:
+
+buildPythonPackage rec {
+  pname = "PlexAPI";
+  version = "3.2.0";
```

Comment:
3.3.0 is not tagged on GitHub, so I just stuck with 3.2.0 for now.

Reply:
you can raise an issue upstream, most release managers generally like to keep on top of stuff like that

Reply:
https://github.com/pkkid/python-plexapi/issues/424

Reply:
pkkid/python-plexapi#424